src/main/scala/firrtl/passes/ReplSeqMem.scala

```diff
@@ -0,0 +1,142 @@
+package firrtl.passes
+
+import com.typesafe.scalalogging.LazyLogging
+import firrtl._
+import firrtl.ir._
+import Annotations._
+import java.io.Writer
+import AnalysisUtils._
+
+sealed trait PassOption
+case object InputConfigFileName extends PassOption
+case object OutputConfigFileName extends PassOption
+case object PassCircuitName extends PassOption
+
+object Error {
+  def apply[T <: Any](msg: String) = throw new Exception(msg)
```

Comment:
Consider using https://github.com/ucb-bar/firrtl/blob/master/src/main/scala/firrtl/passes/Passes.scala#L54 instead? It can collect many errors and throw them together. If this isn't desired, then don't worry about it.